src/witan/app/util.clj

```diff
@@ -34,6 +34,15 @@
                params)))
       true)))
 
+(defn is-a-number? [txt]
```

Comment:
I remember the discussion. Just want to make sure we also catch number with decimals, so ... Double/valueOf ?